webextensions/_locales/fr/messages.json

```diff
@@ -49,32 +49,32 @@
   "tabbar_newTabButton_tooltip": { "message": "Ouvrir un nouvel onglet" },
 
   "tabbar_newTabAction_tooltip":           { "message": "Ouvrir un nouvel onglet comme…" },
-  "tabbar_newTabAction_independent_label": { "message": "Onglet indépendant" },
-  "tabbar_newTabAction_child_label":       { "message": "Onglet enfant" },
-  "tabbar_newTabAction_sibling_label":     { "message": "Onglet au même niveau" },
-  "tabbar_newTabAction_nextSibling_label": { "message": "Onglet au même niveau suivant" },
+  "tabbar_newTabAction_independent_label": { "message": "Onglet &indépendant" },
+  "tabbar_newTabAction_child_label":       { "message": "Onglet &enfant" },
+  "tabbar_newTabAction_sibling_label":     { "message": "Onglet au même &niveau" },
+  "tabbar_newTabAction_nextSibling_label": { "message": "Onglet au même niveau, juste après" },
 
-  "tabbar_newTabWithContexualIdentity_tooltip": { "message": "New Container Tab" },
+  "tabbar_newTabWithContexualIdentity_tooltip": { "message": "Nouvel onglet dans un conteneur" },
   "tabbar_newTabWithContexualIdentity_default": { "message": "Défaut" },
 
-  "tabDragHandle_tree_tearoff_tooltip":  { "message": "Start dragging to detach this tree from the window" },
-  "tabDragHandle_tree_bookmark_tooltip": { "message": "Start dragging to bookmark this tree" },
-  "tabDragHandle_tab_tearoff_tooltip":   { "message": "Start dragging to detach this individual tab from the window" },
-  "tabDragHandle_tab_bookmark_tooltip":  { "message": "Start dragging to bookmark this individual tab" },
+  "tabDragHandle_tree_tearoff_tooltip":  { "message": "Glissez pour détacher cet arbre de la fenêtre" },
+  "tabDragHandle_tree_bookmark_tooltip": { "message": "Glissez pour ajouter cet arbre aux marque-pages" },
+  "tabDragHandle_tab_tearoff_tooltip":   { "message": "Glissez pour détacher cet unique onglet de la fenêtre" },
```

Comment:
`cet onglet particulier`

Reply:
I’m not sure that «particulier» is better than «unique» as opposed to a tree of tabs. As an alternative, I can propose «cet onglet seul».

Reply:
«seulement cet onglet» sounds good to me, what do you think about it?

Reply:
Yes indeed!